plugins/heartbeats/scripts/heartbeats.py

```
@@ -0,0 +1,403 @@
+    """
+    if crontab_file is not None:
+        if not crontab_file.exists():
+            return []
+        try:
+            raw = crontab_file.read_bytes()
+        except OSError as error:
+            raise CrontabError("cannot read %s: %s" % (crontab_file, error))
+        return _to_lines(_decode(raw, str(crontab_file)))
+
+    try:
+        result = subprocess.run(["crontab", "-l"], capture_output=True, check=False)
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)